handbook/engineering/career-development/talent-review-process.md

```diff
@@ -52,10 +53,28 @@ Finally, we should have a [retrospective](../../retrospectives/index.md) after e
 
 Our [career development framework](framework.md) defines the levels that exist at Sourcegraph. Compensation for each of these levels is derived from Option Impact, a startup compensation tool that Sourcegraph and many other companies use to determine their compensation bands.
 
-## Considerations for promotion
+## Promotion requests
+
+Promotion requests are made immediately following the [talent review callibration meeting](#calibration-meeting).
+
+The Engineering Manager will complete a promotion request using the [promotion request template](https://docs.google.com/document/d/1DHyPtIooi0J2426iemzzTqLle9UBasrqLZPGET1O7Sc/edit) and submit it for approval.
+
+Following approval from to the relevant decision-makers (EM, Director, VP), the promotion request is forwarded to the relevant teammates in PeopleOps (refer to [People Ops Communications Matrix](https://docs.google.com/spreadsheets/d/1JItBWbfKV9lr-LAmE19I0JMvu3Cvh0AdrEHDv-r1E2w/edit#gid=0)) for confirmation. Once confirmed by PeopleOps the promotion can be communicated to the teammate.
+
+Decisions regarding promotions are prioritized by the decision-makers, and made quickly. Promotions are communicated back to the ICs in a timely manner.
```

Comment:
In the section below, you include a template for communicating the compensation increase to the IC. Is there a similar (or the same) template for communicating promotions?

Reply:
@chrispine I don't have one actually but good callout, I will create one and add it here.